src/server/protocol.ts

```diff
@@ -15,6 +15,8 @@ namespace ts.server.protocol {
         /* @internal */
         CompletionsFull = "completions-full",
         CompletionDetails = "completionEntryDetails",
+        /* @internal */
+        CompletionDetailsFull = "completionEntryDetailsFull",

```

Comment:
I think the usual convention is `-full`, rather than `Full`.

Reply:
that is an oversight, we should fix it.